mixed-content/generic/mixed-content-test-case.js

```diff
@@ -0,0 +1,224 @@
+function MixedContentTestCase(scenario, description, sanityChecker) {
+  var insecureProtocol = "http:"
```

Comment:
Tiny nit: The protocol is "http". The `:` separates the protocol from the rest of the URL. :)